dev/benchmarks/macrobenchmarks/test_memory/large_images.dart

```diff
@@ -16,20 +16,9 @@ Future<void> endOfAnimation() async {
   } while (SchedulerBinding.instance.hasScheduledFrame);
 }
 
-int iteration = 0;
-
-class LifecycleObserver extends WidgetsBindingObserver {
-  @override
-  void didChangeAppLifecycleState(AppLifecycleState state) {
-    debugPrint('==== MEMORY BENCHMARK ==== $state ====');
-    debugPrint('This was lifecycle event number $iteration in this instance');
```

Comment:
fwiw, this was useful debugging information when examining the test

Reply:
afaik, this life cycle state is only printed when one try to switch between multiple apps, or go back to the home screen. In this specific memory test, we're not doing any of that so in my test this function ends up never been called.

Reply:
maybe this code was copied from another test where it was more useful?

Reply:
Correct. I copied them from `flutter_gallery__back_button_memory` test, and later found that the `READY` message is the only one that I need.